DataScience/Analytics/prototype/lifeEventsAddresses.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""
+ONS Address Index - Life Events Testing
+=======================================
+
+A simple script to test the life events data. In general there are no UPRNs in this data,
+hence the most one can do is the get the linking rate and clerically review whether the
+found matches are correct or not.
+
+This is a prototype code aimed for experimentation and testing. There are not unit tests.
+The code has been written for speed rather than accuracy, it therefore uses fairly aggressive
+blocking. As the final solution will likely use ElasticSearch, the aim of this prototype is
+not the highest accuracy but to quickly test different ideas, which can inform the final
+ElasticSearch solution.
+
+
+Running
+-------
+
+After all requirements are satisfied, the script can be invoked using CPython interpreter::
+
+    python lifeEventsAddresses.py
+
+
+Requirements
+------------
+
+:requires: pandas (0.19.1)
+:requires: addressLinking (and all the requirements within it)
+
+
+Author
+------
+
+:author: Sami Niemi (sami.niemi@valtech.co.uk)
+
+
+Version
+-------
+
+:version: 0.2
+:date: 14-Dec-2016
+"""
+from Analytics.linking import addressLinking
+import pandas as pd
```

Comment:
Didn't know :-)... to my knowledge pd is a common abbreviation for pandas, like np is for numpy, or sp for scipy, so I'll stick to that.